tests/Jobs/ImportRockTheVoteReportTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Tests\Http\Web;
+
+use Tests\TestCase;
+use Chompy\User;
+use Chompy\Jobs\ImportFileRecords;
+use Illuminate\Support\Facades\Bus;
+use Chompy\Models\RockTheVoteReport;
+use Illuminate\Support\Facades\Storage;
+use Chompy\Jobs\ImportRockTheVoteReport;
+
+class ImportRockTheVoteReportTest extends TestCase
+{
+    /**
+     * Test that report is not downloaded when status not complete.
+     *
+     * @return void
+     */
+    public function testReportStatusNotComplete()
+    {
+        Bus::fake();
+
+        $report = factory(RockTheVoteReport::class)->create();
+        $user = User::forceCreate(['role' => 'admin']);
+
+        $this->rockTheVoteMock->shouldReceive('getReportStatusById')->andReturn((object) [
+            'status'=> 'building',
+            'record_count' => 117,
+            'current_index' => 3,
+        ]);
+        $this->rockTheVoteMock->shouldNotReceive('getReportByUrl');
+
+        $importRockTheVoteReportJob = new ImportRockTheVoteReport($user, $report);
+
+        $importRockTheVoteReportJob->handle();
```

Comment:
It might be simpler to use the `::dispatch` syntax to do this in one shot:

```
ImportRockTheVoteReport::dispatch($user, $report);
```

I think manually instantiating & calling `handle` might also bite us if we ever use dependency injection in this job.

Reply:
Could it already be biting us? I had tried calling the job this way, following by example in our [CallPower test](https://github.com/DoSomething/chompy/blob/674ad32ebffcc0bec959f1d63e4fb69fa3bb9579/tests/Http/CallPowerTest.php#L92), but because we're calling `Bus::fake` in this test -- the assertions for verifying the report model update fail:
```
        ImportRockTheVoteReport::dispatch($user, $report);

        $this->assertEquals($report->status, 'building');
```

fails with
```
1) Tests\Http\Web\ImportRockTheVoteReportTest::testReportStatusNotComplete
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'queued'
+'building'
```